lib/cell.rb

```diff
@@ -1,5 +1,12 @@
-class Cell          # The cell class. See also: the element for each column 
-  def empty?        # (for every row) in a Board instance. 
-    true
+class Cell
+  attr_accessor :painted
+  def initialize
+    @painted = false
+  end
+  def empty?
+    @painted == false
+  end
+  def painted?
+    @painted == true
```

Comment:
L7/L10 — you generally don't need to compare boolean values to literal `true` or literal `false`, as the boolean value already has that information. If `x == true` then you can just write `x`.

Also we may want to just get rid of the `empty?` method. We can now write `!cell.painted?` which feels closer to the domain we're modeling.